src/Ucommerce.Sitecore.Cli/Commands/Database.cs

```diff
@@ -40,7 +41,8 @@ public class DbBackup : Database, ICommand
     {
         public ValueTask ExecuteAsync(IConsole console)
         {
-            console.Output.WriteLine("Backing up database..");
+            var logging = new CliLogger(console);
+            logging.Information<DbUpgrade>("Backing up database..");
```

Comment:
Should this be "DbBackup" instead of "DbUpgrade"?

Reply:
You are so right my friend I will fix later